### PR TITLE
feat(crons): Handle disabled monitors in the overview UI

### PR DESCRIPTION
**Pull request by evanpurkhiser**
- Adds a Play / Pause (Activate / Deactivate) button to each row.
 - Indicates disabled monitors via a tag + opacity adjustment

Looks like this

### CODE REVIEW

File: static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx [hunk 1]
```diff
@@ -50,15 +54,42 @@ export function TimelineTableRow({
...
     : monitor.environments.slice(0, MAX_SHOWN_ENVIRONMENTS);
 
+  const isDisabled = monitor.status === 'disabled';
+
+  const monitorDetails = singleMonitorView ? null : (
+    <DetailsArea>
+      <DetailsLink to={`/organizations/${organization.slug}/crons/${monitor.slug}/`}>
+        <DetailsHeadline>
+          <Name>{monitor.name}</Name>
+          {isDisabled && <Tag icon={<IconCheckmark />}>{t('Disabled')}</Tag>}
```
Comment (rjo100) [inline static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx:64]: do we want a `checkmark` here?

Comment (evanpurkhiser) [reply static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx:64]: lol oops, was testing something 😅

File: static/app/views/monitors/components/overviewTimeline/index.tsx [hunk 2]
```diff
@@ -95,6 +95,22 @@ export function OverviewTimeline({monitorList}: Props) {
     });
   };
 
+  const handleToggleStatus = async (monitor: Monitor) => {
+    const data: Partial<Monitor> = {
+      status: monitor.status === 'active' ? 'disabled' : 'active',
+    };
+
+    const resp = await updateMonitor(api, organization.slug, monitor.slug, data);
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/overviewTimeline/index.tsx:103]: Small bug here. When we update we don't pass along the environments filter, so updates come back with ALL environments, which isn't what we actually want.

**Review by rjo100 — APPROVED** [review]

File: static/app/views/monitors/components/overviewTimeline/index.tsx (hunk 2 above)
Comment (davidenwang) [reply static/app/views/monitors/components/overviewTimeline/index.tsx:103]: hmm what happens with this bug, does that mean that all environments will be shown after disabling even if you have only a few selected?

File: static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx [hunk 3]
```diff
@@ -162,6 +229,19 @@ const TimelineRow = styled('div')<{singleMonitorView?: boolean}>`
...
+  &:hover ${DetailsActions} {
+    display: flex;
+  }
+
+  /* Hide trailing items on hover */
+  &:hover ${DetailsHeadline} ${Tag} {
+    visibility: hidden;
+  }
+
+  /* Disabled monitors become more opqaue */
```
Comment (davidenwang) [inline static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx:242]: nit: opaque*

File: static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx [hunk 4]
```diff
@@ -162,6 +229,19 @@ const TimelineRow = styled('div')<{singleMonitorView?: boolean}>`
...
+  --disabled-opacity: ${p => (p.isDisabled ? '0.6' : 'unset')};
```
Comment (davidenwang) [inline static/app/views/monitors/components/overviewTimeline/timelineTableRow.tsx:243]: dang this is pretty neat

**Review by davidenwang — APPROVED** [review]

File: static/app/views/monitors/components/overviewTimeline/index.tsx (hunk 2 above)
Comment (davidenwang) [reply static/app/views/monitors/components/overviewTimeline/index.tsx:103]: yea it might be a little bit more annoying but it's probably better

**Pull request merged**